src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageExtensions.cs

```diff
@@ -102,8 +114,19 @@ public static class LoggerConfigurationAzureTableStorageExtensions
         {
             if (loggerConfiguration == null) throw new ArgumentNullException("loggerConfiguration");
             if (String.IsNullOrEmpty(connectionString)) throw new ArgumentNullException("connectionString");
-            var storageAccount = CloudStorageAccount.Parse(connectionString);
-            return AzureTableStorage(loggerConfiguration, storageAccount, restrictedToMinimumLevel, formatProvider, storageTableName, writeInBatches, period, batchPostingLimit, keyGenerator);
+
+            try
+            {
+                var storageAccount = CloudStorageAccount.Parse(connectionString);
+                return AzureTableStorage(loggerConfiguration, storageAccount, restrictedToMinimumLevel, formatProvider, storageTableName, writeInBatches, period, batchPostingLimit, keyGenerator);
```

Comment:
Both are needed to prevent runtime exceptions as we don't know which constructor will be used. The try/catch here catches any exceptions from the constructor that parses the connection string into a `CloudStorageAccount` variable. However, end users may also call the constructor with an already-created `CloudStorageAccount` instance directly, so a try/catch is needed there as well (as a communication failure will raise an exception).